### DISCUSSION TITLE
Apply BPF Stenographer filter

### DISCUSSION THREAD

**Discussion by commgina** [Security-Onion-Solutions/securityonion#13602] [2.4] [answered]
### Version

2.4.70

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

airgap

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

32

### Storage for /

1TB

### Storage for /nsm

1TB

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I want to apply a BPF filter to avoid receiving traffic from certain networks that come through the sensors. Should I apply this filter to the manager node, or should I create a filter for each node? Will this filter applied to Stenographer prevent alerts from being generated, or will it only prevent PCAPs of those alerts from being stored? 
![image](https://github.com/user-attachments/assets/3cbd355d-29a4-4517-afa5-026af8df5ab1)
Please tell me if the format in the image is correct.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by kvkamin** [comment, marked as answer]
If you apply it as default, it will push out to every sensor, obviously you can select a particular sensor.  According the the text on this page https://docs.securityonion.net/en/2.4/bpf.html 
, your service will not see the traffic, so no PCAP or Alert from that service.  Notice you can apply different rules to PCAP, Suricata, and Zeek.

Also, you might consider using OR instead of AND. 

ie not net A or not net B or not net C.  

like !(net A or net B or net C)

**Reply by commgina** [reply]
Thank you!!!
